### ISSUE TITLE
React 6.32691a crashes when trying to use /re commands behind bungeecord

### ISSUE THREAD

**Issue by zicy** [VolmitSoftware/React#152]
If I in spigot.yml set bungeecord to true and connect thru my Bungee setup and run any /re command then the servere stops responding.

The only plugin i have installed on my test server are React 6.32691a, and running a newly build spigot 1.12.2 version

The setting in spigot.yml
```yml
settings:
  bungeecord: true
```

The [server.log](https://hastebin.com/uqoxivetil)

Spigot version:
> 23:52:12 [INFO] This server is running CraftBukkit version git-Spigot-4bd94dc-c765646 (MC: 1.12.2) (Implementing API version 1.12.2-R0.1-SNAPSHOT)
> 23:52:12 [INFO] Checking version, please wait...
> 23:52:13 [INFO] You are 1 version(s) behind

**Comment by cyberpwnn** [comment]
Can you reproduce this constantly? More specifically, **Are you sure this isnt the server simply running really low on memory?** because i have bungeecord setup with ip forwarding and can use react normally.

**Comment by zicy** [comment]
It can run react 5 without any problems on the same server, it currently have 4GB of ram to play with.
It crashes every time it run any /re command

**Comment by zicy** [comment]
Bungee
[config](https://hastebin.com/lipifukisa)

Spigot
[server.properties](https://hastebin.com/obiwokusen)

**Comment by cyberpwnn** [comment]
Try this on the latest version

**Comment by zicy** [comment]
Okay i have now tested again on version 6.0.42, and it still dies, but after testing more i found out that its not because of Bungee Cord. The reason my server dies are a suddenly high CPU usage when i login...

in plugin/React/worlds/world--8227260175080116856.yml
I have found out that it will not use all of my CPU power if i disable actions in the world config file.

The region i am standing have a lot of hoppers and plenty of redstone, i will try and disable the different action in the config and see which one are the culprit.

This image are when i not started the testserver
![image](https://user-images.githubusercontent.com/3398437/35879073-8a51d7be-0b7a-11e8-93c2-98e16faaccf3.png)

This image are when i have started the testserver and i have logged in
![image](https://user-images.githubusercontent.com/3398437/35879298-1cf8c06e-0b7b-11e8-9460-d6aa85bca847.png)

We use a Intel(R) Xeon(R) CPU E3-1275 v5 @ 3.60GHz CPU

**Comment by zicy** [comment]
Okay when i disable entity in config-experimental.yml i can join,
```
  entity:
    chunk-radius: 2
    chunk-clusterer: 30
    propagate: false
    chunk-multiplier: 2
```

According to the Minecraft debug overlay there are 349 entitys in the area, i will investigate. 
I will try and look for enti

**Comment by zicy** [comment]
Okay i my case i looks like it was armor stands, which in my case are protected by LWC.
It looks like React keep trying to cull them but that React can't remove them.

To reproduce in my case I stacked 31 armor stands which was locked with LWC in the same location, and then subscribed to rai using /re sub rai and could see the trying to cull entitys message over and over again

**Comment by cyberpwnn** [comment]
Interesting. Ill adjust tickrates and improve fail checks

**Comment by cyberpwnn** [comment]
Fixed in 6.0.49
